src/components/Greetings.jsx

```diff
@@ -0,0 +1,22 @@
+
+import React from 'react'
+
+function Greetings({lang, children}) {
+let checkLang;
+ if (lang === "de") {
```

Comment:
Good job! Another way to do this would be using the switch statement, like:

```
switch(lang) {
        case 'en':
            return <h1>Hello {children}</h1>
        case 'es':
            return <h1>Hola {children}</h1>
        case 'fr':
            return <h1>Bonjour {children}</h1>
        case 'de':
            return <h1>Hallo {children}</h1>
        default:
            return <h1>{children}</h1>
    }
```